City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-4078: Enable and configure content lock module, add listing for l…

…ocked units and services as well, add translations

## How to install
- Make sure your instance is up and running on latest dev branch.
- Checkout this branch.
- Run `make drush-cim drush-cr drush-uli`

## How to test
- For the testing instructions, use the same instructions as in this PR: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/254

Also check this PR:
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/254
